content/en/user-guide/tools/localstack-extensions/aws-replicator-extension/index.md

```diff
@@ -1,15 +1,14 @@
 ---
-title: "AWS Replicator Extension"
-linkTitle: "AWS Replicator Extension"
+title: "AWS Replicator"
+linkTitle: "AWS Replicator"
 weight: 1
-description: >
-  A LocalStack extension to replicate AWS resources into your local machine
+description: A LocalStack extension that replicates AWS resources into your LocalStack container
 ---
 
-AWS Replicator Extension is a LocalStack extension that allows you to replicate AWS resources into your local machine.
+AWS Replicator Extension is a LocalStack extension that allows you to replicate AWS resources into your LocalStack container.
```

Comment:
nit: id' go for 'instance' instead of 'container' to indicate the running instance.